pyenvbuilder/main.py

```diff
@@ -0,0 +1,51 @@
+'''
+pyenvbuilder's commands
+'''
+import argparse
+import logging
+from pyenvbuilder import __version__
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def launch(log_level="INFO"):
```

Comment:
For now I just have the `log_level` as an argument but I will be adding more as I develop this.